doc/source/report/ultradian.rst

```diff
@@ -127,27 +124,59 @@ Basic steps::
-      such as the IS/AS condition, the purpose of the active: like
-      food or water and any other important features. Then we can use
-      linear mixed model to get the pattern of the movements in
-      different time period.
-   -  Statsmodels.api package can be used to decompose the dataset we
-      have.
+   The mixed model is frequently used for longitudinal analysis. We should specify the random effects and fixed effects first. Since it is ultradian analysis so we only need to focus on the hour factor and their cycle which we can get from the previous LS test. The random effect is the mouse id. Basically we have 4 different mouses in one strain and we only want to compare the different pattern among these three strains. So if we set the random effect to be mouse id, the effects from different mouses will be cancelled off and we can also test the significance of these effects. The response variable will be one of the six features listed before. After that we can use the mixed model to get the pattern of the movements in different time period.
+
+- Build the model
+
+  Take `Food` feature as an example, and here strain0 means a dummy variable indicates whether the mouse belongs to strain 0 or not. Also  strain1 means a dummy variable indicates whether the mouse belongs to strain 1 or not. The interaction terms means strain0*hour, strain1*hour. We add this because we want to figure out whether the strain and hour have some interaction effect in Food feature. (`i denote ith strain, j denote the jth mouse`)
+
+.. math::
 
-Autocorrelation analysis.
+  Food_{ij} = f(strain0_{ij} , strain1_{ij} , hour_{ij} , cycle_{ij}) + interactions + \beta_j mouse
 
--  Implement numpy.correlate functions to investigate autocorrelation
-   in several variables for features of interest (food, water,
-   distance, active\_state probability ...)
--  Explore different kinds of autocorrelation models (spatial
-   analysis, regression analysis) on variables generated by other
-   subprojects.
+- Perform significance test
+
+  Here we have two purposes, firstly we want to figure out if the effects from different mouses are significant. Secondly we want to figure out if the patterns for different strains are significantly different. To test the first one, we just need to use the t test and get the p value from the result by using the `statsmodels.formula.api` package. For the second one, we can perform the likelihood ratio test on the interaction terms.
+
+Firstly the summary of the full model result is below:(also take the Food feature as an example). Here is the result that we fitted the second degree fucntion and since the cycles from the previous study are very similar between strains so we did not include it here. We can see that the effects of the mouse from the same strain is not significant. However the p value here seems to indicate that the interation terms is not as significant as the other factors. So we consider to conduct likelihood ratio test.
```

Comment:
"First, we look at the summary of the full model :(also take the Food feature as an example). To get this result, we fit the second degree function. Since the cycles from the previous study are very similar between strains, we did not include it here. We can see that the effects of the mouse from the same strain are not significant. However the p value here seems to indicate that the interaction terms is not as significant as the other factors. So we consider conducting the likelihood ratio test."